network/utils.go

```diff
@@ -242,3 +242,33 @@ func isFlagSet(name string, flagset *flag.FlagSet) bool {
 	})
 	return set
 }
+
+// GetDBURLHost return DB host from MySQL/PostgreSQL connection string to use as SNI
+// PostgreSQL - postgresql://{user}:{password}@{host}:{port}/{dbname}
+// MySQL - ({user}:{password}@tcp({host}:{port})/{dbname}
+func GetDBURLHost(connectionString string, useMySQL bool) (string, error) {
```

Comment:
if we parse `connectionString`, then maybe `GetConnectionStringHost` will be more precise? Or `GetDriverConnectionStringHost` to note that it is driver specific...